components/google_cloud/actions/create-bucket/create-bucket.mjs

```diff
@@ -0,0 +1,45 @@
+import googleCloud from "../../google_cloud.app.mjs";
+import storageClasses from "../../utils/storageClasses.mjs";
+
+export default {
+  name: "Create Bucket",
+  version: "0.0.1",
+  key: "google_cloud-create-bucket",
+  type: "action",
+  description: "Creates a bucket on Google Cloud Storage [See the cocs](https://googleapis.dev/nodejs/storage/latest/Bucket.html#create)",
```

Comment:
`cocs` -> `docs`

Reply:
@dylburger can you please approve deployment for this once more?